dask/dataframe/io/tests/test_csv.py

```diff
@@ -1333,14 +1333,7 @@ def test_error_if_sample_is_too_small():
 
 
 def test_read_csv_names_not_none():
-    text = (
```

Comment:
I noticed that in some situations like this one, the ruff version is less readable than the black version. Beyond style, what is the advantage of ruff over black here? 

I know that ruff is faster, but in the dask repo I don't think the linter is a significant amount time.